examples/lib/stories/components/clip_component_example.dart

```diff
@@ -0,0 +1,81 @@
+import 'dart:math';
+import 'dart:ui';
+
+import 'package:flame/components.dart';
+import 'package:flame/effects.dart';
+import 'package:flame/game.dart';
+import 'package:flutter/material.dart' hide Gradient;
+
+class _Rectangle extends RectangleComponent {
+  _Rectangle()
+      : super(
+          size: Vector2(200, 200),
+          anchor: Anchor.center,
+          paint: Paint()
+            ..shader = Gradient.linear(
+              Offset.zero,
+              const Offset(0, 100),
+              [Colors.orange, Colors.blue],
+            ),
+          children: [
+            SequenceEffect(
+              [
+                RotateEffect.by(
+                  pi * 2,
+                  LinearEffectController(.4),
+                ),
+                RotateEffect.by(
+                  0,
+                  LinearEffectController(.4),
+                ),
+              ],
+              infinite: true,
+            ),
+          ],
+        );
+}
+
+class ClipComponentExample extends FlameGame {
+  static String description = '';
+
+  void increaseSize() {
+    descendants().whereType<PositionComponent>().forEach((component) {
+      component.size = component.size * 1.1;
+    });
+  }
+
+  void decreaseSize() {
+    descendants().whereType<PositionComponent>().forEach((component) {
+      component.size = component.size * .9;
+    });
+  }
```

Comment:
These functions look unused, right?

Reply:
Yeah, good catch, I was going to use them but decided otherwise and forgot to remove them.